txts/bans_and_wither_defences.txt

```diff
@@ -0,0 +1,268 @@
+During The Rain of the MPC we deiscoverd many diffrent ways to kick people without being a server owner
+
+**Book Banning**
+
+*Basic Item Ban*
+*Piglin Ban*
+*Fox Ban*
+*Allay Ban*
+*Zone Protection Item*
+
+**Chunk Bans**
+
+*Furnace Ban*
+*Sign Ban*
+*Pot Ban*
+
+**Server Side Ban**
+
+*Paper Memory Ban*
+*Spigot/Vanilla Memory Ban*
+*Sound Surpression Ban*
+*Memory Void*
+
+**Hoax Bans**
+
+*Chest bans*
+*Beacon Ban*
+*Map Ban*
+*Void Ban*
+
+
+
+Article For Item Bans
+
+** Basic Item Bans**
+
+Over time there have been many changes to how bans work and diffrent designs we will focus on the 3 biggest changes and and the best ban mashine
+
+*1.11*
+This version first made item bans posseble (Item bans were discoverd by Jumbo_Warrior in early 2017.) the shulker box made it posseble to have over 2 mb in one item wich is more then servers were willing to send you so you would get kicked. This was done by putting 27 books with 100 pages of high data characters (one like this 􏿿) and then break the shulker so it drops now the nbt data will be send to you and kick in a range of 31.5  you from the server by building a clock with droppes and hoppers resets the 5 minute despawn timer
```

Comment:
Also should probably include the rectangle character as its the most recognised.